### PR TITLE
chore(release): Changelog for release 3.8.0

### PR DESCRIPTION
**Pull request by GMishx**
## Description

Changelog for release 3.8.0

Also, added few badges to the README.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,62 @@
...
+```
+> adityabisoi <adityabisoi1999@gmail.com>
+> Anupam <ag.4ums@gmail.com>
+> Carmen Bianca Bakker <carmenbianca.bakker@liferay.com>
+> Gaurav Mishra <mishra.gaurav@siemens.com>
+> Kaushl2208 <kaushlendrapratap.9837@gmail.com>
+> Mikko Murto <mikko.murto@gmail.com>
+> sahil <sjha200000@gmail.com>
+> Shaheem Azmal M MD <shaheem.azmal@siemens.com>
+> sjha2048 <sjha200000@gmail.com>
```
Comment (ag4ums) [inline CHANGELOG.md:35]: same mail-id (sjha200000@gmail.com ) two different name can we take one.... or need to take both

File: README.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # FOSSology
 
+[![GPL-2.0](https://img.shields.io/github/license/fossology/fossology)](LICENSE)
 [![Travis-CI Build Status](https://travis-ci.org/fossology/fossology.svg?branch=master)](https://travis-ci.org/fossology/fossology)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2395/badge)](https://bestpractices.coreinfrastructure.org/projects/2395)
 [![Coverage Status](https://coveralls.io/repos/github/fossology/fossology/badge.svg?branch=master)](https://coveralls.io/github/fossology/fossology?branch=master)
+[![Slack Channel](https://img.shields.io/badge/slack-fossology-blue.svg?longCache=true&logo=slack)](https://join.slack.com/t/fossology/shared_invite/enQtNzI0OTEzMTk0MjYzLTYyZWQxNDc0N2JiZGU2YmI3YmI1NjE4NDVjOGYxMTVjNGY3Y2MzZmM1OGZmMWI5NTRjMzJlNjExZGU2N2I5NGY)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/fossology/fossology)](https://github.com/fossology/fossology/releases/latest)
```
Comment (ag4ums) [inline README.md:8]: :+1:

**Comment by mcjaeger** [comment]
I think we have the issue with the bulk reuse. If that is true what Julian reported it is a serious thing

File: CHANGELOG.md (hunk 1 above)
Comment (GMishx) [reply CHANGELOG.md:34]: Removed the `sahil` entry as the correct username is @sjha2048

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,64 @@
...
+
+Please check https://archive.softwareheritage.org/api/#rate-limiting for more
+info.
+
+Some notes about the UTF-8 database. The copyright (and sister) agent now
+creates only UTF-8 string. So it is safe to update to Postgres with UTF-8
+encoded database. For more information, please refer to the wikipage
+[Migration to UTF-8 DB](https://github.com/fossology/fossology/wiki/Migration-to-UTF-8-DB)
+
+#### Credits to contributors for 3.8.0-RC1
```
Comment (ag4ums) [inline CHANGELOG.md:22]: release version 3.8.0 from 3.8.0-RC1

Comment (GMishx) [reply CHANGELOG.md]: Done, please check.

**Pull request merged**